src/CanonicalGraphExamples.jl

```diff
@@ -104,3 +103,69 @@ function loadCanonicalFG_CaesarRing1D(;graphinit::Bool=false)
 
   return fg
 end
+
+
+
+"""
+    $SIGNATURES
+Continuous, linear scalar and multivariate test graph generation. Follows a line
+with the pose id equal to the ground truth.
+"""
+function lineStepFG(lineLength::Int;
```

Comment:
Want suggest perhaps staying with the naming convention of `loadCanonical_LIneStepFG` for now, if that is okay please?

Reply:
No problem, Is `load` still correct as it generates a graph from parameters.